tests/test_filtsmooth/test_particlefiltsmooth/test_particle_filter.py

```diff
@@ -81,43 +59,52 @@ def particle(
     return particle
 
 
+@pytest.fixture()
+def regression_problem(problem):
+    """Filter and regression problem."""
+    *_, regression_problem = problem
+
+    return regression_problem
```

Comment:
Note that I am not increasing the `delta_t` here, because this would have to happen where the `RegressionProblem` is instantiated.
(Before: `delta_t = 16 * info["dt"]`

Thus, the test will run for longer.

Reply:
I guess this is resolved?